site/content/docs/concepts/environments.en.md

```diff
@@ -9,14 +9,19 @@ In the diagram below we have an application called _MyApp_ with two services, _A
 ## Creating an Environment
 
 To create a new environment in your app, you can run `copilot env init` from within your workspace. Copilot will ask you what you want to name this environment and what profile you'd like to use to bootstrap the environment. These profiles are AWS [named profiles](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) which are associated with a particular account and region. When you select one of these profiles, your environment will be created in whichever account and region that profile is associated with.
-
-
-```bash
+```console
 $ copilot env init
 ```
 
-After you run `copilot env init` you can watch as Copilot sets up all the environment resources, which can take a few minutes. Once all those resources are created, the environment will be linked back to the application account. This allows actors in the application account to manage the environment, even without access to the environment account. This linking process also creates and configures new regional ECR repositories, if necessary.
+After you run `copilot env init`, you can watch as Copilot sets up the first of the environment's resources. IAM roles and other infrastructural resources are created. If the environment was created with a different AWS account than the app, the environment will be linked back to the application account; this allows actors in the application account to manage the environment, even without access to the environment account. Copilot creates an [env manifest](../manifest/environment.en.md) at `copilot/environments/[env name]/manifest.yml`.
```

Comment:
I think we can be more specific to what `env init` creates. They are not super arbitrary - we probably won't need to create more resources during this step any time soon. What do you think of something along the line:


```suggestion
After you run `copilot env init`, you can watch as Copilot sets up the two IAM roles that are essential in updating and managing the environment. <the rest>`.
```